### DISCUSSION TITLE
"float point error" is reported at the end of injection

### DISCUSSION THREAD

**Discussion by wdmhouston** [GEOS-DEV/GEOS#1688] [General]
@francoishamon,

    Here is a test case with two scenarios: 1 well and 3 wells.   "case_3wells" has 3 wells with a single perforation and geosx reports float point error at the end of injection.  If there is only 1 well in "case_1well", geosx runs fine. 

   Please take a look and fix it in future if it's confirmed as a bug.

[geosx_crash.zip](https://github.com/GEOSX/GEOSX/files/7535139/geosx_crash.zip)

**Comment by paveltomin** [comment]
Runs fine with latest GEOS (after bringing input file up-to-date)
[case_3wells_updates.zip](https://github.com/GEOS-DEV/GEOS/files/13461768/case_3wells_updates.zip)
